IO.Eventuate.Tram.Tests/TestHelpers/TestServicesHost.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using IO.Eventuate.Tram.Local.Kafka.Consumer;
+using IO.Eventuate.Tram.Database;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Hosting;
+
+namespace IO.Eventuate.Tram.Tests.TestHelpers
+{
+    public class TestServicesHost
+    {
+        public string _aggregateType = "test";
+        public void SetAggregateType(string aggregateType)
+        {
+            _aggregateType = aggregateType;
+        }
+        public IHost SetUpHost()
+        {
+            var _host = new HostBuilder()
+           .ConfigureServices((hostContext, services) =>
+           {
+               // Logging 
+               services.AddLogging(builder =>
+               {
+                   builder.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Debug);
+                   builder.AddConsole();
+                   builder.AddDebug();
+               });
+               // DBContext
+               services.AddDbContext<EventuateTramDbContext>((provider, o) =>
```

Comment:
This commit https://github.com/eventuate-examples/eventuate-tram-core-dotnet-examples-basic/commit/9aeea28a26132c5e334d2381c39e4c756b6bd1ec removed `services.AddDbContext<EventuateTramDbContext>`

Reply:
Removed